TheaterDatabase/TheaterDatabase.sqlproj

```diff
@@ -64,25 +64,15 @@
   </ItemGroup>
   <ItemGroup>
     <Build Include="dbo\Tables\Account.sql" />
-    <Build Include="dbo\Tables\CreativeTeamMember.sql" />
     <Build Include="dbo\Tables\GalleryImage.sql" />
-    <Build Include="dbo\Tables\HashTag.sql" />
-    <Build Include="dbo\Tables\HashTag_Performance.sql" />
     <Build Include="dbo\Tables\Performance.sql" />
```

Comment:
same - performances loaded from wordpress site. Why do we still need this table?